StaggeredCoupledProcesses/TH/ThermalDensityDrivenFlow2D/thermal_gravity_driven_flow2d.prj

```diff
@@ -0,0 +1,304 @@
+            <integration_order>2</integration_order>
+
+            <thermal_conductivity>lambda</thermal_conductivity>
+            <heat_capacity>c_p</heat_capacity>
+            <density>rho</density>
+
+            <process_variables>
+                <process_variable>temperature</process_variable>
+            </process_variables>
+
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="heat_flux_x" output_name="heat_flux_x" />
+            </secondary_variables>
+        </process>
+        <process>
+            <name>LiquidFlow</name>
+            <type>LIQUID_FLOW</type>
+            <integration_order>2</integration_order>
+
+            <darcy_gravity>
+               <!-- axis_id: 0, 1, or the dimension of space minus one -->
+               <axis_id>1</axis_id>
+               <!-- g>=0. g=0: non gravity term -->
+               <g>9.81</g>
+            </darcy_gravity>
+
+            <process_variables>
+                <process_variable>pressure</process_variable>
+            </process_variables>
+
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="darcy_velocity_x" output_name="v_x" />
+                <secondary_variable type="static" internal_name="darcy_velocity_y" output_name="v_y" />
+            </secondary_variables>
+
+            <material_property>
+                <fluid>
+                    <density>
+                       <type>TemperatureDependent</type>
+                       <temperature0> 293.0 </temperature0>
+                       <beta> 4.3e-4 </beta>
+                       <rho0>1000.</rho0>
+                    </density>
+                    <viscosity>
+                        <type>Constant</type>
+                        <value> 1.e-3 </value>
+                    </viscosity>
+                    <specific_heat_capacity>
+                       <type>Constant</type>
+                       <value> 4000. </value>
+                    </specific_heat_capacity>
+                    <thermal_conductivity>
+                       <type>Constant</type>
+                       <value> .6 </value>
+                    </thermal_conductivity>
+                </fluid>
+                <porous_medium>
+                    <porous_medium id="0">
+                        <permeability>
+                            <values>1.e-12</values>
+                        </permeability>
+                        <porosity>
+                            <type>Constant</type>
+                            <value> 0.2 </value>
+                        </porosity>
+                        <storage>
+                            <type>Constant</type>
+                            <value> 0.0 </value>
+                        </storage>
+                    </porous_medium>
+                </porous_medium>
+                <solid>
+                    <thermal_expansion>solid_thermal_expansion</thermal_expansion>
```

Comment:
No, this is for solid and it takes the Parameter type data, which is a sort of global data.